.github/workflows/docs.yml

```diff
@@ -1,5 +1,11 @@
 name: Docs
 
+# If a pull-request is pushed then cancel all previously running jobs related
+# to that pull-request
```

Comment:
This looks wrong.

I think the MR needs to be rebased.

Reply:
Yes it does. I fixed it

Reply:
Thanks. Can you squash it down to one commit? Instead of the current commit and then a "merge commit of main branch"

Reply:
yes

Reply:
> yes

Awesome! Thank you.

Reply:
All in one commit. My code and the merge

Reply:
Is the message `feat: add delete on package_file` object ok ?